pkg/controllers/node/termination/terminator/eviction.go

```diff
@@ -171,6 +175,10 @@ func (q *Queue) Reconcile(ctx context.Context) (reconcile.Result, error) {
 // Evict returns true if successful eviction call, and false if there was an eviction-related error
 func (q *Queue) Evict(ctx context.Context, key QueueKey) bool {
 	ctx = log.IntoContext(ctx, log.FromContext(ctx).WithValues("Pod", klog.KRef(key.Namespace, key.Name)))
+	evictionMessage, err := evictionReason(ctx, key, q.kubeClient)
+	if err != nil {
```

Comment:
I personally think we should just silently fail here. I'm worried about super noisy logs in the case where a nodeclaim accidentally goes away, or if there are other issues with the apiserver

Reply:
I hate silently failing, bc when debugging I often end up having to re-add all the logs for suppressed errors. So I compromised and moved it to `V(1)`.